docs/changelog/1392.md

```diff
@@ -0,0 +1 @@
+- Added a new interface file `precice/Version.h`, which provides version macros `PRECICE_VERSION_MAJOR`, `PRECICE_VERSION_MINOR`, `PRECICE_VERSION_PATCH` as well as the stringified version `PRECICE_VERSION`. This file is now included by `SolverInterface.hpp` and `SolverInterfaceC.h`.
```

Comment:
From the user side, the version macro `PRECICE_VERSION_GTE´ should be of practical relevance. We should mention it in the changelog.

Reply:
True. Probably even more relevant than the macros themselves.